web/tendrl/multiuser_cluster_import.rst

```diff
@@ -1,2 +1,55 @@
-multiuser_cluster_import
-*******************************************************************************
\ No newline at end of file
+Multiuser - Cluster import
+****************************
+
+:authors:
+          - mkudlej@redhat.com
+:caseposneg: positive
+
+
+Description
+============
+
+Try to import cluster twice. Testcase is tested with one (or two users) and
+with 2 different browser sessions (two different session IDs). Goal is to try to import
+cluster from *Import wizard*. User(s) should have permission to import cluster.
+
+Setup
+======
+
+#. Prepare environment as described in :doc:`/configurations/multiuser_initial_setup`.
+
+#. Prepare cluster ready for import.
+
+Test Steps
+===========
+
+.. test_action::
+   :step:
+       In both browsers click on *Import Cluster* button.
+   :result:
+       In both browsers *Cluster Import* wizard starts.
```

Comment:
Do you mean that you initiate import of the same cluster two times (almost at the same time)?